lib/mongo/operation/indexes.rb

```diff
@@ -27,26 +27,7 @@ module Operation
     # @since 2.0.0
     class Indexes
       include Specifiable
-
-      # Execute the operation.
-      #
-      # @example
-      #   operation.execute(server)
-      #
-      # @param [ Mongo::Server ] server The server to send the operation to.
-      #
-      # @return [ Mongo::Operation::Indexes::Result ] The operation result.
-      #
-      # @since 2.0.0
-      def execute(server)
-        if server.features.op_msg_enabled?
-          OpMsg.new(spec).execute(server)
-        elsif server.features.list_indexes_enabled?
-          Command.new(spec).execute(server)
-        else
-          Legacy.new(spec).execute(server)
-        end
-      end
+      include OpMsgOrListIndexesCommand
```

Comment:
This appears not to be used anywhere else; is the reason for it to just be consistent about not having any implementations of `#execute` in operation classes?

Reply:
Yes, for consistency with the other ones.

Reply:
Sounds good